deploy/installer/config.cmake

```diff
@@ -0,0 +1,23 @@
+if(WIN32)
+    set(RootDir "@RootDir@")
+    configure_file(
+        ${CMAKE_CURRENT_LIST_DIR}/config/windows.xml.in
```

Comment:
It looks xml files after cmake processing are generated in the source tree. Probably it's not good practice, because we need to cleanup source tree every time, for example if we will need to switch to another branch

Reply:
please check all such occurrences and propose some solution